includes/admin/reporting/reports.php

```diff
@@ -25,7 +25,7 @@ function edd_reports_page() {
 	$active_tab = isset( $_GET['tab'] ) ? $_GET['tab'] : 'reports';
 	?>
 	<div class="wrap">
-		<h2><?php _e( 'Easy Digital Downloads Reports', 'easy-digital-downloads' ); ?></h2>
+		<h2><?php _e( 'Reports', 'easy-digital-downloads' ); ?></h2>
```

Comment:
```suggestion
		<h1><?php esc_html_e( 'Reports', 'easy-digital-downloads' ); ?></h1>
```
Otherwise this page has no `h1` and the text should be escaped anyway.